processor/src/test/java/org/mapstruct/ap/test/namesuggestion/SuggestMostSimilarNameTest.java

```diff
@@ -66,13 +66,21 @@ public void testNameSuggestion() {
             @Diagnostic(type = PersonGarageWrongTargetMapper.class,
                 kind = javax.tools.Diagnostic.Kind.ERROR,
                 line = 19,
-                message = "Unknown property \"garage.colour.rgb\" in result type org.mapstruct.ap.test.namesuggestion" +
-                    ".Person. Did you mean \"garage.color\"?"),
+                message = "Unknown property \"colour\" in type org.mapstruct.ap.test.namesuggestion.Garage for target" +
+                    " name \"garage.colour.rgb\". Did you mean \"garage.color\"?"),
+            @Diagnostic(type = PersonGarageWrongTargetMapper.class,
+                kind = javax.tools.Diagnostic.Kind.WARNING,
```

Comment:
suddenly.. more WARNINGS?

Reply:
Before the entire `TargetReference` for `garage.colour.rgb` was invalid which meant that handling the defined mappings led to an hour which stopped the creating of the `BeanMappingMethod` and this not reporting the unmapped target.

However, with the new change the target reference does not have `valid` property anymore, so the defined mappings would not report this error and thus we would reach the unmapped target properties reporting. 

IMO this is a good thing, since you have more warnings / errors that you can look at initially